eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -132,11 +132,9 @@ jobs:
             - Windows.81.Amd64.Open
             - Windows.10.Amd64.ServerRS5.Open
             - Windows.10.Amd64.Server19H1.Open
-            - Windows.10.Amd64.Server20H2.Open
```

Comment:
This one is not Server Core, are these failures happening on this SKU as well?

Reply:
Yup, sorry, these are in 20H2 non-Core, typo

Reply:
@safern Anything else?